source/reference/verification/verifier.txt

```diff
@@ -0,0 +1,120 @@
+.. _c2c-migration-verifier:
+
+==============================
+Verify with Migration Verifier
+==============================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Migration Verifier connects to the source and destination clusters and performs
+a series of verification checks, comparing documents, views, and indexes to
+confirm the sync was successful.
+
+Unlike other verification methods, Migration Verifier can run concurrent with
+``mongosync``, checking documents on the destination cluster as they sync. This
+can reduce the amount of time needed to verify the success of the sync before
+transferring your application load from the source to the destination cluster.
+
+About This Task
+---------------
+
+Migration Verifier is an experimental and unsupported tool.
+
+For installation instructions, see
+`GitHub <https://github.com/mongodb-labs/migration-verifier>`__.
+
+Steps
+-----
+
+.. procedure::
+
+   .. step:: Sync Clusters
+
+      Run :program:`mongosync` to sync the source and destination clusters.
+
+   .. step:: Start Migration Verifier
+
+      After you start ``mongosync``, start the ``migration-verifier`` process:
+
+      .. code-block:: bash
+
+         migration-verifier --verifyAll \
+            --srcURI example.net:27020 \
+            --destURI example.net:27021 \
+            --metaURI example.net:27017
+
+   .. step:: Start the Verification Checks
+
+      To start the verification checks, use cURL to send the ``check`` command to
```

Comment:
I still think this should clarify that migration verifier should be started once mongosync reaches CEA. Currently it implies you can send the check command anytime after mongosync is started.

Reply:
Fixed